Build 909 for 7.17 with status FAILURE - TestInput/Test_first_event

## :broken_heart: Tests Failed


<!-- BUILD BADGES-->
> _the below badges are clickable and redirect to their specific view in the CI or DOCS_
[![Pipeline View](https://img.shields.io/badge/pipeline-pipeline%20-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2F7.17/detail/7.17/909//pipeline) [![Test View](https://img.shields.io/badge/test-test-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2F7.17/detail/7.17/909//tests) [![Changes](https://img.shields.io/badge/changes-changes-green)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2F7.17/detail/7.17/909//changes) [![Artifacts](https://img.shields.io/badge/artifacts-artifacts-yellow)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2F7.17/detail/7.17/909//artifacts) [![preview](https://img.shields.io/badge/docs-preview-yellowgreen)](http://beats_null.docs-preview.app.elstc.co/diff)  [![preview](https://img.shields.io/badge/elastic-observability-blue)](https://ci-stats.elastic.co/app/apm/services/beats-ci/transactions/view?rangeFrom=2023-11-19T03:20:02.250Z&rangeTo=2023-11-19T03:40:02.250Z&transactionName=BUILD+Beats%2Fbeats%2F7.17&transactionType=job&latencyAggregationType=avg&traceId=98a06080a7a87c9f7437f12c8a02a197&transactionId=329210fc70b0763a)

<!-- BUILD SUMMARY-->
<details><summary>Expand to view the summary</summary>
<p>

#### Build stats




* Start Time: 2023-11-19T03:30:02.250+0000


* Duration: 21 min 54 sec




#### Test stats :test_tube:

| Test         | Results                         |
| ------------ | :-----------------------------: |
| Failed       | 2  |
| Passed       | 851  |
| Skipped      | 7 |
| Total        | 860   |


</p>
</details>

<!-- TEST RESULTS IF ANY-->

  ### Test errors [![2](https://img.shields.io/badge/2%20-red)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2F7.17/detail/7.17/909//tests)
<details><summary>Expand to view the tests failures</summary><p>
    
    
##### `TestInput/Test_first_event – github.com/elastic/beats/v7/x-pack/filebeat/input/httpjson/internal/v2`


<ul>
<details><summary>Expand to view the error details</summary><p>

```
 Failed 
 ```
</p></details>
<details><summary>Expand to view the stacktrace</summary><p>

```
 === RUN   TestInput/Test_first_event
    input_test.go:454: 
        	Error Trace:	/var/lib/jenkins/workspace/7.17-909-1da57593-bca8-4ae6-9ec4-dad7e786868c/src/github.com/elastic/beats/x-pack/filebeat/input/httpjson/internal/v2/input_test.go:454
        	Error:      	Not equal: 
        	            	expected: map[string]interface {}{"first":"a", "foo":"b"}
        	            	actual  : map[string]interface {}{"first":"none", "foo":"b"}
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,3 +1,3 @@
        	            	 (map[string]interface {}) (len=2) {
        	            	- (string) (len=5) "first": (string) (len=1) "a",
        	            	+ (string) (len=5) "first": (string) (len=4) "none",
        	            	  (string) (len=3) "foo": (string) (len=1) "b"
        	Test:       	TestInput/Test_first_event
--- FAIL: TestInput/Test_first_event (2.01s)
 
 ```
</p></details>
</ul>
    
##### `TestInput – github.com/elastic/beats/v7/x-pack/filebeat/input/httpjson/internal/v2`


<ul>
<details><summary>Expand to view the error details</summary><p>

```
 Failed 
 ```
</p></details>
<details><summary>Expand to view the stacktrace</summary><p>

```
 === RUN   TestInput
--- FAIL: TestInput (7.22s)
 
 ```
</p></details>
</ul>
    
</p></details>


<!-- STEPS ERRORS IF ANY -->














### Steps errors [![4](https://img.shields.io/badge/4%20-red)](https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats%2F7.17/detail/7.17/909//pipeline)
  <details><summary>Expand to view the steps failures</summary>
  <p>


  
  
  
  
##### `x-pack/filebeat-cloud - mage build test`
<ul>
<li>Took 4 min 5 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/7.17/runs/909/steps/2810/log/?start=0">here</a></li>
<li>Description: <code>mage build test</code></l1>
</ul>
  
  
  
  
##### `Terraform Apply on x-pack/metricbeat/module/aws`
<ul>
<li>Took 4 min 3 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/7.17/runs/909/steps/2600/log/?start=0">here</a></li>
<li>Description: <code>terraform apply -auto-approve</code></l1>
</ul>
  
  
  
  
##### `Terraform Apply on x-pack/metricbeat/module/aws`
<ul>
<li>Took 0 min 23 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/7.17/runs/909/steps/2827/log/?start=0">here</a></li>
<li>Description: <code>terraform apply -auto-approve</code></l1>
</ul>
  
  
  
  
##### `Error signal`
<ul>
<li>Took 0 min 0 sec . View more details <a href="https://beats-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/Beats/pipelines/beats/pipelines/7.17/runs/909/steps/2846/log/?start=0">here</a></li>
<li>Description: <code>Error "hudson.AbortException: script returned exit code 1"</code></l1>
</ul>
  
  </p>
  </details>

--- a/CHANGELOG-developer.next.asciidoc
+++ b/CHANGELOG-developer.next.asciidoc
@@ -63,6 +63,7 @@ The list below covers the major changes between 7.0.0-rc2 and master only.
 - Avoid panicking in `add_fields` processor when input event.Fields is a nil map. {pull}28219[28219]
 - Fix type mismatch in libbeat/metric/system/cgroup/cgv2 when building on mips platforms. {pull}34658[34658]
 - Make winlogbeat/sys/wineventlog follow the unsafe.Pointer rules. {pull}36650[36650]
+- Skip flakey test in HTTPJSON input {issue}34929[34929] {issue}37146[37146] {pull}37162[37162]
 
 ==== Added
 

--- a/x-pack/filebeat/input/httpjson/internal/v2/input_test.go
+++ b/x-pack/filebeat/input/httpjson/internal/v2/input_test.go
@@ -30,6 +30,8 @@ func TestInput(t *testing.T) {
 		baseConfig  map[string]interface{}
 		handler     http.HandlerFunc
 		expected    []string
+
+		skipReason string
 	}{
 		{
 			name:        "Test simple GET request",
@@ -246,6 +248,8 @@ func TestInput(t *testing.T) {
 			},
 		},
 		{
+			skipReason: "flakey test - see https://github.com/elastic/beats/issues/34929",
+
 			name: "Test first event",
 			setupServer: func(t *testing.T, h http.HandlerFunc, config map[string]interface{}) {
 				registerPaginationTransforms()
@@ -407,6 +411,10 @@ func TestInput(t *testing.T) {
 	for _, testCase := range testCases {
 		tc := testCase
 		t.Run(tc.name, func(t *testing.T) {
+			if tc.skipReason != "" {
+				t.Skipf("skip: %s", tc.skipReason)
+			}
+
 			tc.setupServer(t, tc.handler, tc.baseConfig)
 
 			cfg := common.MustNewConfigFrom(tc.baseConfig)

